Blatt_3/Checksum.java

```diff
@@ -0,0 +1,37 @@
+package Blatt_3;
```

Comment:
we currently don't use packages

Reply:
or use  package Blatt_3.Aufgabe_1;

Reply:
I use Blatt_3.Aufgabe_1 in the next commit.